addons/stock/views/stock_location_views.xml

```diff
@@ -39,6 +39,11 @@
                             <field name="scrap_location" attrs="{'invisible': [('usage', 'not in', ('inventory', 'internal'))]}"/>
                             <field name="return_location"/>
                         </group>
+                        <group string="Cyclic Inventory" attrs="{'invisible': ['|', ('usage', 'not in', ('internal', 'transit')), ('company_id', '=', False)]}">
+                            <field name="cyclic_inventory_frequency"/>
+                            <field name="last_inventory_date"/>
+                            <field name="next_inventory_date" attrs="{'invisible': [('active', '=', False)]}"/>
```

Comment:
I don't get all the `active` related code you made. Couldn't it be left to the orm ?

Reply:
ok I got it :)